lib/formats/odata.js

```diff
@@ -226,9 +226,12 @@ const edmxTemplater = template(`<?xml version="1.0" encoding="UTF-8"?>
 const typeMap = {
   int: 'Edm.Int64',
   decimal: 'Edm.Decimal',
+  boolean: 'Edm.Boolean',
   geopoint: 'Edm.GeographyPoint',
   geotrace: 'Edm.GeographyLineString',
   geoshape: 'Edm.GeographyPolygon',
+  date: 'Edm.Date',
+  time: 'Edm.TimeOfDay',
```

Comment:
While using Power BI, @lognaturel encountered the following error, which seems related:

> OLE DB or ODBC error: [DataSource.Error] OData: Cannot convert the literal ‘09:04:00.000Z’ to the expected type ‘Edm.TimeOfDay’.

The form used was: https://test.getodk.cloud/#/projects/149/forms/all-widgets/submissions

Reply:
oh huh. why is there a tz?

Reply:
https://www.w3.org/TR/xmlschema-2/#time

Reply:
interesting. i didn't know anyone did time w tz. last i understood times w tz are meaningless bc you need the date to actually determine what that time means? either way odata doesn't like it so i guess we'll go back to doing timeofday as string.

Reply:
Agreed it's not a great idea and I try to steer clear of this data type myself. We've previously discussed introducing a new type that can't have a timezone but it hasn't been a priority. Using string for that one seems great.